src/vorta/borg/create.py

```diff
@@ -32,7 +32,10 @@ def process_result(self, result):
                 repo.total_unique_chunks = stats['total_unique_chunks']
                 repo.save()
 
-            self.app.backup_progress_event.emit(self.tr('Backup finished.'))
+            if result['returncode'] == 1:
+                self.app.backup_progress_event.emit(self.tr('Backup finished. Some files were not accessible.'))
```

Comment:
I'm not sure that this is correct, because doesn't borg use returncode=1 as a catch-all for a variety of errors?

Reply:
You're right. Could also be "Backup finished with warnings. See logs for details."

Upside is it's more "correct". Downside is that the large majority of warnings will be access errors. Can't think of other warnings I've seen. That fact is now hidden in the logs.

Reply:
I've added a separate issue #952  to nag Mac users about Full Disk access. This should make such errors very rare and we can use the vague sentence. "Backup finished with warnings. See logs for details."

OK?

Reply:
I've now merged the feature to nag Mac users about proper permissions. Mostly due to a overwhelming response on Twitter: https://twitter.com/borg_base/status/1383302657045979144

With this access errors would be fairly rare everywhere and the more vague sentence is likely ok.